docroot/modules/custom/va_gov_backend/src/Service/UserPermsService.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace Drupal\va_gov_backend\Service;
+
+use Drupal\Core\Session\AccountInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\user\Entity\User;
+use Drupal\workbench_access\Entity\AccessSchemeInterface;
+use Drupal\Core\Access\AccessResult;
+
+/**
+ * For exporting User access status to service.
+ */
+class UserPermsService {
+
+  /**
+   * The active user.
+   *
+   * @var object
+   *  The user object.
+   */
+  private $currentUser;
+
+  /**
+   * The entity interface.
+   *
+   * @var object
+   *  The entity interface object.
+   */
+  private $entityInterface;
+
+  /**
+   * UserPermsService constructor.
+   */
+  public function __construct(AccountInterface $currentUser, EntityTypeManagerInterface $entityInterface) {
+    $this->currentUser = $currentUser;
+    $this->entityInterface = $entityInterface;
+  }
+
+  /**
+   * Getter setter for a, user, defaults to current user.
+   *
+   * @param int $user_id
+   *   A user id to lookup the user. (optional)
+   *
+   * @return \Drupal\Core\Session\AccountInterface
+   *   The user specified by $user_id or the current user
+   */
+  private function getUser($user_id = NULL) {
+    if (!empty($user_id)) {
+      // Not available, so load it.
+      return User::load($user_id);
+    }
+    else {
+      return $this->currentUser;
+    }
+  }
+
+  /**
+   * Check which values are allowed and put into array.
+   *
+   * @param array $form
+   *   The entity add or edit form.
+   * @param array $targets
+   *   The form fields to target for filtering.
+   *
+   * @return array
+   *   The options that are allowed for user selection.
+   */
+  public function userOptionsStorage(array &$form, array $targets) {
+
+    $cid = 'user-dropdown-access-' . $this->getUser()->id();
```

Comment:
Should include  namespace for what is creating it.
```suggestion
    $cid = 'userpermservice-dropdown-access-' . $this->getUser()->id();
```

Reply:
:+1: